lib/screens/SiteLogsScreen.dart

```diff
@@ -64,6 +65,37 @@ class _SiteLogsScreenState extends State<SiteLogsScreen> {
     );
   }
 
+  Widget _buildTextWrapToggle() {
+    return Platform.isIOS
```

Comment:
We seem to have a bit of a convention that platform ternaries use `isAndroid` instead of `isIOS`.  I don't think it really matters, but we definitely have more android checks than ios, and maybe it's worth sticking to that convention.